lambda-functions/notification-template-formatter-service/src/main/java/com/paladincloud/FetchNotificationSettings.java

```diff
@@ -154,6 +157,31 @@ public String handleRequest(Map<String,Object> event, Context context)
         return response;
     }
 
+    private String buildNotificationPlainTextMail(String mailBody, Map<String, Object> messageContentMap, String source) {
+            mailBody=mailBody.replace("${source}",source);
+            mailBody=mailBody.replace("${message}",messageContentMap.get("message").toString());
+            StringBuilder buf = new StringBuilder();
+        for (Map.Entry<String, Object> permission:messageContentMap.entrySet()) {
+            if(permission.getKey().startsWith("permission"))
+            buf.append("<tr><td class=\"rowkey\">")
+                    .append(makeItReadable(permission.getKey()))
+                    .append("</td><td>")
+                    .append(permission.getValue())
+                    .append("</td></tr>");
+        }
+        mailBody=mailBody.replace("${permissionIssues}",buf.toString());
+        return mailBody;
+    }
+
+    private String makeItReadable(String key) {
```

Comment:
Make what readable?

Reply:
When we send request in JSON format , there is no space in the key which results into unreadable format.
This method makes content readble as mentioned in screenshot below
![image](https://github.com/PaladinCloud/CE/assets/103987424/da300c8d-5f53-46cc-9a2e-6c7ba65edb1b)

Reply:
I resolved my comment to move conversation here.
so what we are getting is `permissionIssueForPHDInAccount123412341234` right?
If the purpose to only replace that type of message with something readable - I would extract all digits using regex and then instead of replacing every word, just Created my own String const with proper message:
```
String PERMISSION_ISSUE_MESSAGE = "Permission issue for PHD in account %d";
```
and then use it when it's needed.
Do we have any other message examples which are not covered by this approach?

Reply:
the target type and account keep changing

Examples :
permissionIssueForKMSInAccount123412341234
permissionIssueForPHDInAccount123412341234
permissionIssueForS3InAccount123412341234